package.json

```diff
@@ -88,6 +88,7 @@
     "@metamask/detect-provider": "^1.2.0",
     "@react-spring/web": "^9.5.2",
     "@reduxjs/toolkit": "^1.9.3",
+    "@sentry-internal/browser-utils": "^8.0.0-alpha.8",
```

Comment:
do we need an alpha build for this or can we use beta or rc?
e.g [8.0.0-rc.2](https://www.npmjs.com/package/@sentry-internal/browser-utils/v/8.0.0-rc.2)

Reply:
`^8.0.0-alpha.8` is the one that's automagically installed when running `yarn add`. Though the browser packages uses rc.1 which means we probably should use *at least* that one, and that rc.2 should be a step forward here. c135e50dde